### ISSUE TITLE
Add "Clear" to DatePicker field (aka Add "me" button)

### ISSUE THREAD

**Issue by mikezimm** [mikezimm/TrackMyTime7#86]
When using React Date picker https://github.com/pnp/sp-dev-fx-controls-react/tree/master/src/controls/dateTimePicker

Once you have a date selected, you can not clear/unselect it.

Make a button or something to "Clear" the date.

**Comment by mikezimm** [comment]
Playing with css.... Need to set this css to make date picker full width:

![image](https://user-images.githubusercontent.com/49648086/84546111-8d2d0600-acce-11ea-87ab-21a026eda56d.png)

**Comment by mikezimm** [comment]
Currently looks like this with DatePicker not at 100%

![image](https://user-images.githubusercontent.com/49648086/84546254-f01e9d00-acce-11ea-987b-49aee090ee26.png)

**Comment by mikezimm** [comment]
Making this mod seems to do the trick:
node_modules\@pnp\spfx-controls-react\lib\controls\dateTimePicker\DateTimePicker.module.css
basically set container width to 100%

```
.dateTimePicker_b2e59b80 .container_b2e59b80{display:-ms-flexbox;display:flex;-ms-flex-direction:column;flex-direction:column; width:100%}
```

**Comment by mikezimm** [comment]
Note this also has the adverse effect of changing the field alignment in the stack:

Compare Status to CompletedDate
![image](https://user-images.githubusercontent.com/49648086/84546665-cdd94f00-accf-11ea-83d6-beaaafa5d939.png)

**Comment by mikezimm** [comment]
Closed with PR #89
